ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -674,6 +674,15 @@ ompi_mtl_ofi_component_init(bool enable_progress_threads,
 
     hints->domain_attr->resource_mgmt    = FI_RM_ENABLED;
 
+    /*
+     * Libfabric providers choose whether to perform CUDA memory transfers when
+     * FI_HMEM_CUDA_ENABLE_XFER is unspecified. Some providers may choose to
+     * disable CUDA memory transfers by default. Explicitly enabling this option
+     * allows providers to perform CUDA memory transfers via FI_HMEM if
+     * supported.
+     */
+    setenv("FI_HMEM_CUDA_ENABLE_XFER", "1", 1);
```

Comment:
This isn't the only place libfabric is initialized. Look at the OFI BTL as well

Reply:
Also can you put this before the "no_hmem" line? This env doesn't matter when not requesting hmem support

Reply:
ack

Reply:
> This isn't the only place libfabric is initialized. Look at the OFI BTL as well

FWICT, there are 3 files where Libfabric can be initialized:

- `ompi/mca/mtl/ofi/mtl_ofi_component.c`
- `opal/mca/btl/ofi/btl_ofi_component.c`
- `opal/mca/btl/usnic/btl_usnic_component.c`

@wckzhang Would you be against a new wrapper function in `opal/mca/common/ofi/common_ofi.c` around `fi_getinfo()` so we can avoid copy-pasting the `setenv()`/`unsetenv()` sequence in multiple files?